src/shared/components/query/styles.module.scss

```diff
@@ -192,7 +197,8 @@ div.SelectedStudiesWindow {
   }
 
   div.cancerTypeListContainer {
-    overflow: auto;
+    overflow-y: auto;
+    overflow-x: hidden;

```

Comment:
This is to hide horizontal scrollbar, which is visible in IE (see screenshot above).